run_model-indep_with-ml-cats_limits.sh

```diff
@@ -0,0 +1,142 @@
+# based on https://github.com/KIT-CMS/MSSMvsSMRun2Legacy/tree/ntuple_processor
+ulimit -s unlimited
+
+TAG=$1
+MODE=$2
+if [[ $TAG == "auto" ]]; then
+    TAG="cmb_ind"
+fi
+
+
+defaultdir=analysis/$TAG
+analysis="bsm-model-indep"
+sub_analysis="hSM-in-bg"
+categorization="with-sm-ml"
+sm_like_hists="bsm"
+[[ ! -d ${defaultdir} ]] && mkdir -p ${defaultdir}
+[[ ! -d ${defaultdir}/logs ]] && mkdir -p ${defaultdir}/logs
+[[ ! -d ${defaultdir}/limits/condor ]] && mkdir -p ${defaultdir}/limits/condor
+[[ ! -d ${defaultdir}/limits_ind/condor ]] && mkdir -p ${defaultdir}/limits_ind/condor
+defaultdir=$(readlink -f analysis/$TAG)
+datacarddir=${defaultdir}/datacards_${analysis}
+taskname="${analysis}_${TAG}_1"
+taskname2="${analysis}_${TAG}_2"
+
+if [[ $MODE == "initial" ]]; then
+    ############
+    # morphing
+    ############
+    morph_parallel.py --output ${defaultdir}/datacards \
+        --analysis ${analysis} \
+        --sub-analysis ${sub_analysis} \
+        --categorization ${categorization} \
+        --sm-like-hists ${sm_like_hists} \
+        --eras 2016,2017,2018 \
+        --category-list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/sm_neuralnet_categories.txt \
+        --variable nnscore \
+        --sm-gg-fractions ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/data/higgs_pt_reweighting_fullRun2_v2.root \
+        --sm \
+        --parallel 10 2>&1 | tee -a ${defaultdir}/logs/morph_sm_log.txt
+
+    morph_parallel.py --output ${defaultdir}/datacards \
+        --analysis ${analysis} \
+        --sub-analysis ${sub_analysis} \
+        --categorization ${categorization} \
+        --sm-like-hists ${sm_like_hists} \
+        --eras 2016,2017,2018 \
+        --category-list ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/input/mssm_new_categories.txt \
+        --variable mt_tot_puppi \
+        --sm-gg-fractions ${CMSSW_BASE}/src/CombineHarvester/MSSMvsSMRun2Legacy/data/higgs_pt_reweighting_fullRun2_v2.root \
+        --parallel 10 2>&1 | tee -a ${defaultdir}/logs/morph_mssm_log.txt
+
+    ############
+    # combining outputs
+    ############
+    mkdir -p ${datacarddir}/combined/cmb/
+    rsync -av --progress ${datacarddir}/201?/htt_*/* ${datacarddir}/combined/cmb/ 2>&1 | tee -a ${defaultdir}/logs/copy_datacards.txt
+
+elif [[ $MODE == "ws" ]]; then
+    ############
+    # workspace creation
+    ############
+
+    combineTool.py -M T2W -o "ws.root" \
+    -P HiggsAnalysis.CombinedLimit.PhysicsModel:multiSignalModel \
+    --PO '"map=^.*/ggh_(i|t|b).?$:r_ggH[0,0,200]"' \
+    --PO '"map=^.*/bbh$:r_bbH[0,0,200]"' \
+    -i ${datacarddir}/combined/cmb/ \
+    -m 125.0 --parallel 4 | tee -a ${defaultdir}/logs/workspace_independent.txt
+
+    ############
+    # job setup creation
+    ############
+    cd ${defaultdir}/limits_ind/condor
```

Comment:
This would override results from the model-independent limits with the classic categories if they ran beforehand, right?

Reply:
I case the tag is choses properly not.

Reply:
but yes, in case of "auto" flag it would be an overwrite. I'll correct that

Reply:
solved